script/Ciel/Day 12/12_01_CIEL12_2B.txt

```diff
@@ -374,7 +374,7 @@ I feel like I'm always the one waiting for you to come to me,
 -- Page 54, Offset 31001.
 -- 「それはそれは。嬉しいですけど、遠野くん、わたしのクラスを知らないでしょ？　３年のＢクラスですから覚えておいてくださいね」
 "Well, isn't that something?
- I'm happy to hear that, but you don't know which class I'm in, do you Tohno-kun?
+ I'm happy to hear that, but you don't know which class I'm in, do you, Tohno-kun?
```

Comment:
In the context of the other one, should this comma be removed as well?

Reply:
Saying this one aloud it seems OK to me